rubicon_ml/viz/metric_lists_comparison.py

```diff
@@ -154,7 +154,7 @@ def update_metric_heatmap(*args):
 
             data_array = np.array(heatmap_data)
             numerator = data_array - data_array.min(axis=0)
-            denominator = data_array.max(axis=0) - data_array.min(axis=0)
+            denominator = np.array(data_array.max(axis=0) - data_array.min(axis=0))
```

Comment:
same as above unless there's some specific reason this needs to be changed

Reply:
I was running into a couple errors before that this change fixed but I tried it again and it seemed to work now without this change.